DirectProgramming/DPC++FPGA/include/pipe_utils.hpp

```diff
@@ -121,12 +121,14 @@ struct write_currying<WriteFunc, BaseTy, std::index_sequence<I...>> {
   }
 };
 
-}  // namespace
+}  // namespace detail
 
 // =============================================================
 // PipeArray
 // =============================================================
 
+namespace fpga_tools {
```

Comment:
Should we put the detail namespace inside fpga_tools (i.e. move this to the top of the file)?  Not sure what the standard pattern is here.

Reply:
Yes. Done.